rllib/core/rl_trainer/tf/tf_rl_trainer.py

```diff
@@ -162,9 +159,9 @@ def apply_gradients(self, gradients: Dict[ParamRef, TensorType]) -> None:
             optim.apply_gradients(zip(gradient_list, variable_list))
 
     @override(RLTrainer)
-    def _make_distributed(self) -> RLModule:
-        # TODO: Does strategy has to be an attribute here? if so it's very hidden to
-        # the user of this class that there is such an attribute.
+    def _make_distributed(self) -> MultiAgentRLModule:
```

Comment:
Rename this method to `_make_distributed_module` for clarity: Otherwise, I as a user would think that the RLModule is already there (as a non-distributed) and we just convert it to a distributed one. OR another assumption would be that this method makes the RLTrainer itself distributed (in some way).

Reply:
fxed.